bulk_site_creator/views.py

```diff
@@ -104,6 +102,7 @@ def job_detail(request, job_id):
 @require_http_methods(["GET", "POST"])
 def new_job(request):
     sis_account_id = request.LTI["custom_canvas_account_sis_id"]
+    print(f'SIS Account ID: {sis_account_id}')
```

Comment:
Would we want these to be logging statements?

Reply:
Had a couple of debug print statements, those are removed in the latest commit